boards/FSM/Src/App/App_SetPeriodicCanSignals.c

```diff
@@ -17,3 +18,34 @@ void App_SetPeriodicCanSignals_SecondaryFlowMeter(
     App_CanTx_SetPeriodicSignal_SECONDARY_FLOW_RATE(
         can_tx, App_FlowMeter_GetFlowRate(flow_meter));
 }
+
+void App_SetPeriodicCanSignals_LeftWheelSpeedSensor(
+    struct FsmCanTxInterface *can_tx,
+    struct WheelSpeedSensor * wheel_speed_sensor)
+{
+    App_CanTx_SetPeriodicSignal_LEFT_WHEEL_SPEED(
+        can_tx, App_WheelSpeedSensor_GetWheelSpeed(wheel_speed_sensor));
+}
+
+void App_SetPeriodicCanSignals_RightWheelSpeedSensor(
+    struct FsmCanTxInterface *can_tx,
+    struct WheelSpeedSensor * wheel_speed_sensor)
+{
+    App_CanTx_SetPeriodicSignal_RIGHT_WHEEL_SPEED(
```

Comment:
Not point to make a function if it's just one line like this

Reply:
I removed the SetPeriodicSignal fxn and called the CanTx_SetPeriodicSignal function directly in the state machine